x-pack/legacy/plugins/canvas/i18n/functions/dict/demodata.ts

```diff
@@ -13,7 +13,7 @@ import { DemoRows } from '../../../canvas_plugin_src/functions/server/demodata/g
 export const help: FunctionHelp<FunctionFactory<typeof demodata>> = {
   help: i18n.translate('xpack.canvas.functions.demodataHelpText', {
     defaultMessage:
-      'A mock data set that includes project {ci} times with usernames, countries, and run phases.',
+      'A sample data set that includes project {ci} times with usernames, countries, and run phases.',
```

Comment:
This is the help message for the datasource.  I'm not sure if this is actually displayed anywhere.  Should we update the text that is actually displayed (i18n/ui/DemoData)?

Reply:
I updated in the 18n/ui/DemoData, but also changed "mock" across all usages per Kaarina's comment to use "sample" vs "mock"